Examples/MAX32655/I2C_Sensor_ADT7420/main.c

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file        main.c
+ * @brief       I2C generic sensor Example
+ * @details     This example uses the I2C Master to read/write from/to the I2C sensor.
+ */
+
+/******************************************************************************
+ *
+ * Copyright (C) 2022-2023 Maxim Integrated Products, Inc. All Rights Reserved.
+ * (now owned by Analog Devices, Inc.),

```

Comment:
You can remove the Maxim Integrated copyright line (9-10) since this is a new project.

Reply:
Hi @sihyung-maxim , thank you for your review. I removed these two lines.